CountItems/CountWorker.js

```diff
@@ -17,6 +17,35 @@ class CountWorker {
         this.handleMessage = this.handleMessage.bind(this);
     }
 
+    _serializeBigInts(obj) {
```

Comment:
The idea is that in s3utils we have a master-worker approach, where workers send their results to the master. This process serializes the results; yet the master has a need to know what values are bigint, and what are strings.

As such, these functions (and the equivalent ones in the master process class) will automatically use a special field, `__bigint`, to dynamically detect that an object is in fact a bigint.

Reply:
@benzekrimaha we might want to dedup this, with some static functions in a util file